src/main/java/com/findmyrecycling/fmrenterprise/dto/Facility.java

```diff
@@ -2,6 +2,14 @@
 
 import lombok.Data;
 
+/**
+ * Represents facility data
```

Comment:
Add basic Javadoc to facility DTO

Reply:
Some of this can be given by the variable names themselves.  Consider adding JavaDoc where you can describe the purpose of the method and its parameters, which goes beyond the names themselves.